### PR TITLE
Add return types to a few fetch* methods.

### PR DESCRIPTION
**Pull request by blair**
Add return types to `fetchOHLCVC`, `fetchOrderStatus`, `fetchUnifiedOrder`, `fetchMarkOHLCV`, `fetchIndexOHLCV` and `fetchPremiumIndexOHLCV`.

### CODE REVIEW

File: ts/src/base/Exchange.ts [hunk 1]
```diff
@@ -3187,12 +3187,12 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchOrder() is not supported yet');
     }
 
-    async fetchOrderStatus (id: string, symbol: string = undefined, params = {}) {
+    async fetchOrderStatus (id: string, symbol: string = undefined, params = {}): Promise<Order['status']> {
```
Comment (carlosmiei) [inline ts/src/base/Exchange.ts:3190]: Can we keep `Promise<string>` for now?

Comment (blair) [reply ts/src/base/Exchange.ts:3190]: Sure. Can I add a line to remind us in the future?

`// TODO: update Promise<string> with Promise<Order['status']>`

Out of curiosity, what's blocking the change to use `Promise<Order['status']>?

**Comment by blair** [comment]
I updated the PR to switch to `Promise<string>`.

Please take another look.

**Comment by blair** [comment]
BTW, Order looks like this already, so it already has a `string` in it.  

```
export interface Order {
...    status: 'open' | 'closed' | 'canceled' | string;
```

Is the request for `Promise<string>` for transpiling?

File: ts/src/base/Exchange.ts (hunk 1 above)
Comment (carlosmiei) [reply ts/src/base/Exchange.ts]: > BTW, Order looks like this already, so it already has a `string` in it.
> 
> ```
> export interface Order {
> ...    status: 'open' | 'closed' | 'canceled' | string;
> ```
> 
> Is the request for `Promise<string>` for transpiling?

@blair Yes, we have to take into consideration all languages when making a change

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**